selendroid-standalone/src/main/java/io/selendroid/standalone/android/impl/AbstractDevice.java

```diff
@@ -547,16 +547,15 @@ public String getExternalStoragePath() {
   /** {@inheritdoc} */
   public String getCrashLog() {
     String crashLogFileName = ExternalStorageFile.APP_CRASH_LOG.toString();
-    File crashLogFile = new File(getExternalStoragePath(), crashLogFileName);
```

Comment:
Removed the use of the File object since the actual file it should be referencing is on the device while the File object itself is on the host computer. In other words, it was just being used for modifying strings and was causing problems on Windows.